hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -1388,6 +1268,30 @@ public static void assertCompletesEventually(String message, ProgressCheckerTask
     public static void assertTrueEventually(AssertTask task) {
         assertTrueEventually(null, task, ASSERT_TRUE_EVENTUALLY_TIMEOUT);
     }
+    public static void assertFalseEventually(AssertTask task, long timeoutSeconds) {
```

Comment:
minor: we can add a newline here

Reply:
01ed2d56c20bfe1a50c755e38ea8f45c146d6e2e